public/js/admin/login.js

```diff
@@ -0,0 +1,61 @@
+// selecting elements from the html page
+const {
+  form, emailInput, passwordInput, loginBtn,
+} = querySelectors(
+  ['form', 'emailInput', 'passwordInput', 'loginBtn'],
+  ['.form', '.email', '.password', '.login'],
+);
+
+// this function shows an error message to the user
+const showErrorMsg = (form, button, msg) => {
+  const errorMsg = document.createElement('span');
+  errorMsg.appendChild(document.createTextNode(msg));
```

Comment:
@ahmedisam99 Another alternative to this line is `errorMessage.textContent = msg;`